app/javascript/hooks/queries/recordings/useRecordingsReSync.jsx

```diff
@@ -7,7 +7,7 @@ export default function useRecordingsReSync() {
 
   return useQuery(
     'getRecordingsResync',
-    () => axios.get('/recordings/resync.json').then(() => queryClient.invalidateQueries('getRecordings')),
+    () => axios.get('/recordings/resync.json').then(() => queryClient.invalidateQueries('getServerRecordings')),
```

Comment:
Sorry for the late reply.
Just to note, I think we should do the cache invalidation in an `onSuccess` callback and maybe make this `resync` **API** more of a `POST` request because its causes a side effect and not fetches data.